src/main/webapp/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "llvm-dashboard",
+  "version": "1.0.0",
+  "description": "a dashboard for viewing the build status of buildbots for the llvm project",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "parcel index.html",
+    "build": "parcel build index.html"
+  },
+  "author": "",
```

Comment:
All Graph Geckos are authors of this project.

Reply:
Should I list that as Graph Geckos, or as our names respectivelly?

Reply:
I had a look around, and it seems that package.json's author field is only supposed to be used for a single author. Do you suggest I write my own name there, or "Graph Geckos"?

Reply:
Found a field called contributors - I'll use that instead.